Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Scripts/blades/export-settings.js

```diff
@@ -3,16 +3,22 @@ angular.module('virtoCommerce.exportModule')
         var blade = $scope.blade;
         blade.canStartProcess = false;
         blade.isLoading = true;
-        blade.exportDataRequest = {};
+        //blade.exportDataRequest = {};
         blade.isExportedTypeSelected = false;
         
         function initializeBlade() {
             exportApi.getProviders(function (result) {
                 if (result && result.length) {
                     blade.allProviders = result;
                     fillProviders();
+                    if (typeof (blade.exportDataRequest.defaultProvider) !== 'undefined') {
+                        blade.selectedProvider = _.find(blade.providers,
+                            function (item) { return item.name === blade.exportDataRequest.defaultProvider});
+                    }
                 }
             });
+
+            blade.isExportedTypeSelected = typeof (blade.exportDataRequest.exportTypeName) !== 'undefined';
```

Comment:
Thus, having non empty `blade.exportDataRequest` we could move this line to line 7